### PR TITLE
feat(font): add mononoki (@guillermodotn).

### PR DESCRIPTION
**Pull request by guillermodotn**
### Description

This PR adds the **Mononoki** font as proposed on discussion #5576, to the list of built-in fonts that Monkeytype ships with.

### Checks

- [ ] Adding quotes?
  - [ ] Make sure to include translations for the quotes in the description (or another comment) so we can verify their content.
- [ ] Adding a language or a theme?
  - [ ] If is a language, did you edit `_list.json`, `_groups.json` and add `languages.json`?
  - [ ] If is a theme, did you add the theme.css?
    - Also please add a screenshot of the theme, it would be extra awesome if you do so!
- [ ] Check if any open issues are related to this PR; if so, be sure to tag them below.
- [x] Make sure the PR title follows the Conventional Commits standard. (https://www.conventionalcommits.org for more info)
- [x] Make sure to include your GitHub username inside parentheses at the end of the PR title

### CODE REVIEW

File: frontend/src/styles/fonts.scss [hunk 1]
```diff
@@ -230,3 +230,12 @@
   font-display: block;
   src: url("/webfonts/CommitMono-400-Regular.otf") format("opentype");
 }
+
+@font-face {
+  font-family: "Mononoki";
+  font-style: normal;
+  font-weight: 400;
+  font-display: block;
+  src: url("/webfonts/mononoki-Regular.ttf") format("truetype"),
```
Comment (fehmer) [inline frontend/src/styles/fonts.scss:239]: do we need both? Shouldn't `otf` be the better choice?

**Comment by guillermodotn** [comment]
I will suggest the following two modifications `monkeytype/frontend/src/styles/fonts.scss`:

1. A comment stating the `src` preference, like:

```scss
// In the case that both .otf and .ttf font sources are available,
// we choose the .otf format over the .ttf format due to its advanced 
// typographic features and better support for modern typesetting.
```

2. Delete the `.ttf` src for the font `Boom` if it is not needed, to simplify the file. 
Refer to [fonts.scss#L182](https://github.com/monkeytypegame/monkeytype/blob/aac44aead339cfd69c36e5a75a2e3945a44dd3d5/frontend/src/styles/fonts.scss#L182)

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/9827688646

File: frontend/src/styles/fonts.scss (hunk 1 above)
Comment (guillermodotn) [reply frontend/src/styles/fonts.scss]: I agree with you, let's stick to `.otf` src.

File: frontend/static/about/contributors.json
Comment (Miodec) [inline frontend/static/about/contributors.json:1]: No need for this, contributors are generatec automatically and ordered by the number of contributions. Please revert.

File: frontend/static/webfonts/mononoki-Regular.otf
Comment (Miodec) [inline frontend/static/webfonts/mononoki-Regular.otf:1]: Rename to `Mononoki-Regular.otf`

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by Miodec** [comment]
Also, please switch to `woff` if possible (we are changing everything to `woff` in #5582)

**Comment by Miodec** [comment]
Nevermind sorry, @fehmer already added Mononoki in his PR!

**Comment by fehmer** [comment]
hi @guillermodotn . I'm sorry. I was updating the fonts to use woff2 and there was a misunderstanding with mio. I thought your PR is going to be merged first.

**Pull request closed without merging**